Nautilus/Crafting/ModCraftTreeLinkingNode.cs

```diff
@@ -61,6 +61,9 @@ public ModCraftTreeTab AddTabNode(string nameID, string displayText, Sprite spri
     /// <summary>
     /// Creates a new tab node for the crafting tree and links it to the calling node.
     /// </summary>
+    /// <remarks>
+    /// Please not that this method will NOT set the language lines for the node and you must do it yourself.
```

Comment:
Minor spelling mistake
```suggestion
    /// Please note that this method will NOT set the language lines for the node and you must do it yourself.
```